cg/services/order_validation_service/workflows/metagenome/models/sample.py

```diff
@@ -4,7 +4,7 @@
 
 
 class MetagenomeSample(Sample):
-    concentration_ng_ul: float | None = None
```

Comment:
Why this change? @islean Should this be updated in all sample models?

Reply:
This change seems incorrect! Good catch! I get confused by the different concentration fields (which are similar but different) we have spread throughout our order types